GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -187,7 +187,8 @@ class ChatViewModel: EngagementViewModel, ViewModel {
 
         switch state {
         case .enqueueing:
-            guard !shouldSkipEnqueueingState else { return }
+            guard chatType != .secureTranscript else { return }
+//            guard !shouldSkipEnqueueingState else { return }
```

Comment:
I guess this can be removed

Reply:
fixed [87d33ac](https://github.com/salemove/ios-sdk-widgets/pull/743/commits/87d33ac788cc218a9053491f3ea0e2a7b64c0383)